libs/application/template-api-modules/src/lib/modules/shared/data-providers/user-profile/user-profile.service.ts

```diff
@@ -1,45 +1,135 @@
 import { Auth, AuthMiddleware } from '@island.is/auth-nest-tools'
-import { UserProfileApi } from '@island.is/clients/user-profile'
-import { ProblemError } from '@island.is/nest/problem'
-import { ProblemType } from '@island.is/shared/problem'
-import { isRunningOnEnvironment } from '@island.is/shared/utils'
+import { UserProfileApi, UserProfile } from '@island.is/clients/user-profile'
+import { IslykillService } from '@island.is/api/domains/user-profile'
+import { logger } from '@island.is/logging'
 import { Injectable } from '@nestjs/common'
 import { SharedTemplateApiService } from '../..'
 import { TemplateApiModuleActionProps } from '../../../../types'
+import { ApolloError } from 'apollo-server-express'
```

Comment:
I think we can safely remove the Appollo Gql dependencies in here.

Reply:
Done